app/components/map-form.js

```diff
@@ -171,6 +181,28 @@ export default class MapFormComponent extends Component {
     };
   }
 
+  // union all geometries together, draw a 600 foot buffer around the union
+  @computed('model.bufferSize')
+  get projectGeometryBuffer() {
```

Comment:
Hmm I don't quite understand why this should be moved into the component - is there something preventing you from keeping it there? It looks like all of inputs are related to a model somewhere at some point.